bundles/org.openhab.binding.sensibo/src/main/java/org/openhab/binding/sensibo/internal/handler/SensiboSkyHandler.java

```diff
@@ -167,16 +160,20 @@ protected void handleCommand(final ChannelUID channelUID, final Command command,
                     }
                 } else if (CHANNEL_MODE.equals(channelUID.getId())) {
                     if (command instanceof RefreshType) {
-                        updateState(channelUID, new StringType(unit.getAcState().getMode()));
+                        if (unit.getAcState().isPresent()) {
```

Comment:
Too bad we still need Java8. Perfect fit for `.ifPresentOrElse`... :(

Reply:
I'll fix it when OH moves to JDK11